### PR TITLE
Duplicate Boise, Idaho records

### PR DESCRIPTION
**Pull request by nvkelso**
The point is bunk from Natural Earth. The polygon is good from Quattroshapes.

### CODE REVIEW

File: data/114/190/933/3/1141909333.geojson [hunk 1]
```diff
@@ -2,572 +2,576 @@
...
+    "geom:latitude": 43.60859,
+    "geom:longitude": -116.22749,
+    "iso:country": "US",
+    "lbl:bbox": "-116.24748987,43.5885901073,-116.20748987,43.6285901073",
+    "lbl:latitude": 43.60859,
+    "lbl:longitude": -116.22749,
+    "lbl:max_zoom": 18,
+    "lbl:min_zoom": 13.5,
+    "mz:hierarchy_label": 1,
+    "mz:is_current": 0,
```
Comment (nvkelso) [inline data/114/190/933/3/1141909333.geojson:20]: The Natural Earth record was a point without any descendents, so marking it as not current here.

File: data/114/190/933/3/1141909333.geojson [hunk 2]
```diff
@@ -2,572 +2,576 @@
...
-    "lbl:max_zoom":18.0,
-    "lbl:min_zoom":13.5,
-    "mz:hierarchy_label":1,
-    "mz:is_current":1,
-    "mz:min_zoom":12.0,
-    "name:afr_x_preferred":[
-        "Boise"
+    "edtf:cessation": "",
+    "edtf:inception": "",
+    "edtf:superseded": "2024-05-30",
```
Comment (nvkelso) [inline data/114/190/933/3/1141909333.geojson:7]: Superseded into the Quattroshapes polygon feature today.

File: data/114/190/933/3/1141909333.geojson [hunk 3]
```diff
@@ -2,572 +2,576 @@
...
+      }
+    ],
+    "wof:id": 1141909333,
+    "wof:lastmodified": 1717104892,
+    "wof:name": "Boise",
+    "wof:parent_id": 102086087,
+    "wof:placetype": "locality",
+    "wof:repo": "whosonfirst-data-admin-us",
+    "wof:superseded_by": [
+      85937923
```
Comment (nvkelso) [inline data/114/190/933/3/1141909333.geojson:565]: Superseded into the Quattroshapes polygon feature ID.

File: data/859/379/23/85937923.geojson [hunk 4]
```diff
@@ -2,617 +2,620 @@
...
+    ],
+    "wof:id": 85937923,
+    "wof:lastmodified": 1717104889,
+    "wof:name": "Boise",
+    "wof:parent_id": 102086087,
+    "wof:placetype": "locality",
+    "wof:repo": "whosonfirst-data-admin-us",
+    "wof:superseded_by": [],
+    "wof:supersedes": [
+      1141909333
```
Comment (nvkelso) [inline data/859/379/23/85937923.geojson:610]: Supersedes the Natural Earth point feature.

**Comment by nvkelso** [comment]
Exploratory (1% partial) work towards https://github.com/whosonfirst-data/whosonfirst-data/issues/1732

File: data/114/190/933/3/1141909333.geojson (hunk 2 above)
Comment (stepps00) [reply data/114/190/933/3/1141909333.geojson:7]: When setting the is_current to 0 and superseding, we'd typically also set the cessation or deprecated date, too.. the superseded date is optional (though fine to keep in)

I'd suggest adding the second date before merging.

**Review by stepps00 — APPROVED** [review]
Nit about the edtf dates, but LGTM otherwise.

File: data/114/190/933/3/1141909333.geojson (hunk 2 above)
Comment (nvkelso) [reply data/114/190/933/3/1141909333.geojson:8]: Fixed in https://github.com/whosonfirst-data/whosonfirst-data-admin-us/pull/168/commits/91ea1a458d788b396f3f6ab09c6a700c29e6f877, thanks!

**Pull request merged**